carrot.html

```diff
@@ -24,8 +24,105 @@
     <div>
       <!-- 지우 -->
     </div>
-    <div>
-      <!-- 세안 -->
+    <div class="sean-whole-part">
+      <div class="sean-popular-products">
+        <div>
+          <h1 class="sean-title">중고거래 인기매물</h1>
+        </div>
+        <div class="sean-products-container">
+          <article class="sean-product-whole">
+            <div>
+              <img class="sean-product-image" src="./images/product1.jpg" />
+            </div>
```

Comment:
```suggestion
            <img class="sean-product-image" src="./images/product1.jpg" />
```

위와 동일한 이유로 감싸고 있는 div는 삭제해도 좋을 것 같습니다!